src/components/contacts/ContactList.vue

```diff
@@ -23,7 +26,59 @@ export default {
   },
   props: ['contacts', 'contactClick'],
   data () {
-    return {}
+    return {
+      contactsState: this.setInitialContactsState()
```

Comment:
This won't update when the `contacts` prop changes. I would instead just record the `activeAddress`, and then change:

```js
 :class="{ active: address === activeAddress}"
```

or equivalent
```